README.md

```diff
@@ -17,9 +17,14 @@ Add this to your Gemfile:
 Usage
 =====
 
-    class Topic < ActiveRecord::Base
-       ignore_columns :attributes, :class
-    end
+```ruby
+class Topic < ActiveRecord::Base
+   # IMPORTANT the ignore_columns invocation must go after any table_name_prefix
+   self.table_name_prefix = 'some_prefix'
```

Comment:
I don't think it is clear that you don't need a `table_name_prefix` to make `ignore_columns` work.  Can you have two examples?

Also, can you change the column names to something that is less confusing?  Otherwise, users will think they need to specify the class name as the second parameter to `ignore_columns`.